Qube TechRadar.csv

```diff
@@ -1,6 +1,6 @@
 name,ring,quadrant,isNew,description
 Go,Adopt,languages-and-frameworks,FALSE,"Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
-Visual Studio Code,Trial,Tools,FALSE,"Visual Studio Code is a code editor redefined and optimized for building and debugging modern web and cloud applications."
+Visual Studio Code,Adopt,Tools,TRUE,"Visual Studio Code is an opensource, lightweight, cross-platform code editor for most programming languages & files, with amazing extensions and built-in space for terminals"
```

Comment:
```suggestion
Visual Studio Code,Adopt,Tools,TRUE,"Visual Studio Code is an open-source, cross-platform code editor for most programming languages & files, with amazing extensions and built-in space for terminals"
```

Reply:
Although it has vastly improved, I think better not to say lightweight.

Reply:
Okay, I can live with that..  :D